src/styles/theme/themeConfig.ts

```diff
@@ -0,0 +1,203 @@
+// Base colors - updates should match css vars!
+const PURE_WHITE = "#ffffff";
+const DARK = "#141219";
+const DARK_THREE = "#25222e";
+const DARK_FOUR = "#2f2a3c";
+const OFF_WHITE = "#f7f7f7";
+const WHITE_SIX = "#e7e7e7";
+const WHITE_THREE = "#d8d8d8";
+const BABY_PURPLE = "#b59ff6";
+const LIGHT_PURPLE = "#e7e4f2";
+const CHARCOAL_GRAY = "#3b3649";
+const ALLEN_PURPLE = "#8d87aa";
+const GRAYISH_BROWN = "#4a4a4a";
+const BLUE = "#0094FF";
+const WARM_GRAY = "#979797";
+const TRANSPARENT_WHITE = "rgba(255, 255, 255, 0.65)";
+const TRANSPARENT = "transparent";
+
+const APP_BG_COLOR = OFF_WHITE;
+const TABS_INTERACTION_COLOR = DARK_FOUR;
+const SLIDER_COLOR = WHITE_THREE;
+const CHECKBOX_BG_COLOR = WHITE_SIX;
```

Comment:
Nit: just call this `LAYOUT_BG` and use it for `colorBgBase` and `colorBgLayout` below. or don't have both of those in your light theme and just set them in the actual theme.